app-admin/syslog-ng/syslog-ng-4.3.1.ebuild

```diff
@@ -0,0 +1,186 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
```

Comment:
please drop python 3.9 :)

`PYTHON_COMPAT=( python3_{10..11} )`

Also, if you can, please test if it works with python3_12